lib/u3d/commands_generator.rb

```diff
@@ -148,6 +148,26 @@ def run
         end
       end
 
+      command :dependencies do |c|
+        c.syntax = 'u3d dependencies [-p | --package_manager <package manager>]'
+        c.summary = 'Installs Unity dependencies. [Linux only]'
+        c.description = %(
+#{c.summary}
+
+Regarding the package manager: if none is specified, u3d will try to assume it by testing which one.s is.are installed.
+Only apt-get and yum are tested right now.
+
+If several package manager are installed on your machine simultaneously, we recommend that you specify the one you want to use.
+
+Regarding the dependencies themselves: only dependencies for the editor are installed. WebGL, Android and Tizen require others that you will have to install manually.
```

Comment:
will that work if user doesn't specify -p "apt-get -y" ?

I.e. do we manage input properly?

Reply:
Not working properly right now, I'm correcting that